gnocchi/rest/api.py

```diff
@@ -1967,6 +1974,11 @@ def get(details=True):
         report_dict = {"storage": {"summary": report['summary']}}
         if 'details' in report:
             report_dict["storage"]["measures_to_process"] = report['details']
+        report_dict['metricd'] = {}
+        if members_req:
+            report_dict['metricd']['processors'] = members_req.get()
```

Comment:
tbh, i thought `processors` meant total core count. is there a more standard name we've been using? agent? daemon? or are we using `processor`  name going forward?

Reply:
```class MetricProcessor(MetricProcessBase):
    name = "processing"
    GROUP_ID = b"gnocchi-processing"
```

I'd suggest we change name to processor in gnocchi.cli.metricd too then.